Gruntfile.js

```diff
@@ -55,41 +57,26 @@ module.exports = function(grunt) {
       },
     },
 
-    "closure-compiler": {
-      dist: {
-        files: {
-          "dist/browser-polyfill.min.js": ["dist/browser-polyfill.js"],
-        },
-        options: {
-          // Closure currently supports only whitespace and comment stripping
-          // when both the input and output languages are ES6.
-          compilation_level: "WHITESPACE_ONLY",
-          language_in: "ECMASCRIPT6_STRICT",
-          language_out: "ECMASCRIPT6",
-          output_wrapper: `${LICENSE}\n%output%`,
-        },
+    shell: {
+      minify: {
+        command: "babili dist/browser-polyfill.js -o dist/browser-polyfill.min.js --minified --no-comments --source-maps true",
```

Comment:
Please use grunt-babel for this instead, and ideally set up a minify env in `package.json`